schemas/activity-stream/events/events.1.schema.json

```
@@ -8,6 +8,10 @@
     "addon_version": {
       "type": "string"
     },
+    "browser_session_id": {
+      "description": "A mirror of the browser sessionId, as defined in main/main.4.schema.json",
```

```suggestion
      "description": "A mirror of the browser sessionId, as defined in https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/schemas/telemetry/main/main.4.schema.json",
```

We are now publishing descriptions in the BQ schemas, so this will be viewable in the BQ console. It likely makes sense to give a full URL, since this relative path won't make sense in the BQ console context.

---

Thanks, will add it in.